common/docker/configuration.go

```diff
@@ -0,0 +1,335 @@
+			PortBindings: nat.PortMap{
+				"7349/tcp": []nat.PortBinding{{HostPort: "7349"}},
+				"7350/tcp": []nat.PortBinding{{HostPort: "7350"}},
+				"7351/tcp": []nat.PortBinding{{HostPort: "7351"}},
+			},
+			RestartPolicy: container.RestartPolicy{Name: "unless-stopped"},
+			NetworkMode:   container.NetworkMode(cfg.DockerEnv["CARDINAL_NAMESPACE"]),
+		},
+		Platform: &ocispec.Platform{
+			Architecture: "amd64",
+			OS:           "linux",
+		},
+	}
+}
+
+func NakamaDB(cfg *config.Config) Config {
+	return Config{
+		Name: getNakamaDBContainerName(cfg),
+		Config: &container.Config{
+			Image: "cockroachdb/cockroach:latest-v23.1",
+			Cmd:   []string{"start-single-node", "--insecure", "--store=attrs=ssd,path=/var/lib/cockroach/,size=20%"},
+			Env: []string{
+				"COCKROACH_DATABASE=nakama",
+				"COCKROACH_USER=root",
+				fmt.Sprintf("COCKROACH_PASSWORD=%s", cfg.DockerEnv["DB_PASSWORD"]),
+			},
+			ExposedPorts: nat.PortSet{
+				"26257/tcp": struct{}{},
+				"8080/tcp":  struct{}{},
+			},
+			Healthcheck: &container.HealthConfig{
+				Test:     []string{"CMD", "curl", "-f", "http://localhost:8080/health?ready=1"},
+				Interval: 3 * time.Second, //nolint:gomnd
+				Timeout:  3 * time.Second, //nolint:gomnd
+				Retries:  5,               //nolint:gomnd
+			},
+		},
+		HostConfig: &container.HostConfig{
+			PortBindings: nat.PortMap{
+				"26257/tcp": []nat.PortBinding{{HostPort: "26257"}},
+				"8080/tcp":  []nat.PortBinding{{HostPort: "8080"}},
+			},
+			RestartPolicy: container.RestartPolicy{Name: "unless-stopped"},
+			Mounts: []mount.Mount{{Type: mount.TypeVolume, Source: cfg.DockerEnv["CARDINAL_NAMESPACE"],
+				Target: "/var/lib/cockroach"}},
+			NetworkMode: container.NetworkMode(cfg.DockerEnv["CARDINAL_NAMESPACE"]),
+		},
+	}
+}
+
+func CelestiaDevNet(cfg *config.Config) Config {
+	return Config{
+		Name: getCelestiaDevNetContainerName(cfg),
+		Config: &container.Config{
+			Image: "ghcr.io/rollkit/local-celestia-devnet:latest",
+			ExposedPorts: nat.PortSet{
+				"26657/tcp": struct{}{},
+				"26658/tcp": struct{}{},
+				"26659/tcp": struct{}{},
+				"9090/tcp":  struct{}{},
+			},
+			Healthcheck: &container.HealthConfig{
+				Test:     []string{"CMD", "curl", "-f", "http://127.0.0.1:26659/head"},
+				Interval: 1 * time.Second,
+				Timeout:  1 * time.Second,
+				Retries:  20, //nolint:gomnd
+			},
+		},
+		HostConfig: &container.HostConfig{
+			PortBindings: nat.PortMap{
+				"26657/tcp": []nat.PortBinding{{HostPort: "26657"}},
+				"26658/tcp": []nat.PortBinding{{HostPort: "26658"}},
+				"26659/tcp": []nat.PortBinding{{HostPort: "26659"}},
+				"9090/tcp":  []nat.PortBinding{{HostPort: "9090"}},
+			},
+			RestartPolicy: container.RestartPolicy{Name: "on-failure"},
+			NetworkMode:   container.NetworkMode(cfg.DockerEnv["CARDINAL_NAMESPACE"]),
+		},
+	}
+}
+
+func EVM(cfg *config.Config) Config {
+	daBaseURL := cfg.DockerEnv["DA_BASE_URL"]
+	if daBaseURL == "" {
+		daBaseURL = "http://celestia-devnet"
```

Comment:
The default value here should be `getCelestiaDevNetContainerName(cfg)` i think

Reply:
changed hardcoded `http://celestia-devnet` to `getCelestiaDevNetContainerName(cfg)`